inc/saiswitch.h

```diff
@@ -1269,6 +1269,30 @@ typedef enum _sai_switch_attr_t
     SAI_SWITCH_ATTR_ACL_STAGE_EGRESS,
 
     /**
+     * @brief Max number of Segments in SID List supported
+     *
+     * @type sai_uint32_t
+     * @flags READ_ONLY
+     */
+    SAI_SWITCH_ATTR_SEGMENTROUTE_MAX_SEGMENTS,
+
+    /**
+     * @brief Max number of SID Lists supported
+     *
+     * @type sai_uint32_t
+     * @flags READ_ONLY
+     */
+    SAI_SWITCH_ATTR_SEGMENTROUTE_MAX_SIDLISTS,
+
```

Comment:
I'll remove this for now.  My original idea was for the transit case where multiple SR headers can be appended and this would be the MAX # of headers a vendor could parse.  But due to confusion and folks not ready for SR header stack, I'll just remove this.  We can add if there is demand.